.travis.yml

```diff
@@ -10,7 +10,10 @@ before_install:
 matrix:
   fast_finish: true
 env:
-  - TEST_TYPE=bazel.release
```

Comment:
just realized there is one more place (or more) where this needs to be replaced. Won't bother doing unless people want to go this route.